doc/api/fs.markdown

```diff
@@ -528,13 +530,21 @@ If `options` is a string, then it specifies the encoding. Example:
 
     fs.writeFile('message.txt', 'Hello Node.js', 'utf8', callback);
 
-## fs.writeFileSync(filename, data[, options])
+Any specified file descriptor has to support writing.
```

Comment:
Mind adding that data written to fd will be appended?

Guess we don't expose an `lseek()` in fs so users couldn't reposition where data is written. Hm. Future feature.